oe_corporate_site_info.post_update.php

```diff
@@ -45,3 +45,17 @@ function oe_corporate_site_info_post_update_00002(): void {
     'corporate_body' => 'http://publications.europa.eu/resource/authority/corporate-body',
   ]);
 }
+
+/**
+ * Install required modules for the Accessibility statement field.
+ */
+function oe_corporate_site_info_post_update_00003(): void {
+  $module_handler = \Drupal::service('module_handler');
+  $module_installer = \Drupal::service('module_installer');
+  if (!$module_handler->moduleExists('link')) {
+    $module_installer->install(['link']);
+  }
+  if (!$module_handler->moduleExists('config_translation')) {
+    $module_installer->install(['config_translation']);
```

Comment:
This is wrong. How can we install the config_translation module on every site? What if a site is not translatable? Which languages would they translate to? Just because we make something translatable it doesn't mean we need to force translatability on the site.

Reply:
We subscribe to an event provided by `config_translation` so it kinda needs to be required. Our module cannot be installed\used without `config_translation`.